{{ cookiecutter.repo_name }}/app/src/main/java/{{ cookiecutter.package_dir }}/data/model/DummyEntity.kt

```diff
@@ -1,4 +1,4 @@
-package {{ cookiecutter.package_name }}.model
+package {{ cookiecutter.package_name }}.data.model
```

Comment:
@priyankj-simform  Let's rename the directory from `model` to `entity` and move it inside the `local` package.